### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by andrewdes**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -43,18 +43,19 @@ database operations and takes care of database vendor-specific differences. This
...
+and consumed by other components in the microservice. An Embedded Derby database is used 
+as a data store for all the events.
 
 You will use JPA annotations to define an entity class whose fields are persisted to the 
-database. The interaction between your application and the database is mediated by the persistence 
+database. The interaction between your service and the database is mediated by the persistence 
 context that is managed by an entity manager. In a Java EE environment, one type of entity
-manager is a container-managed entity manager. In this guide you will use dependency injection
-to acquire a container-managed entity manager, placing the burden of opening and closing 
+manager is a container-managed entity manager. In this guide you will use a container-managed
```
Comment (proubatsis) [inline README.adoc]: This makes me want to leave the guide and Google "types of java ee entity managers".

Comment (proubatsis) [reply README.adoc]: It also makes me wonder "what exactly is an entity manager and why is it necessary?"

**Comment by evelinec** [comment]
@proubatsis can you peer review this PR and see everything is good. Thanks.

**Comment by evelinec** [comment]
Merged, so the latest content can be refreshed on the staging site.

**Pull request merged**